### DISCUSSION TITLE
Cannot find module 'langium'

### DISCUSSION THREAD

**Discussion by ballcoach12** [eclipse-langium/langium#1169] [Q&A] [answered]
It  might be just me,  but when I switched today from langium 1.3.0 to 2.0.0, VS Code began complaining that it `Cannot find module 'langium'` .  

Am I the only one who is seeing this?

**Comment by ballcoach12** [comment, marked as answer]
Just saw  the module format changes described in https://github.com/eclipse-langium/langium/blob/main/packages/langium/CHANGELOG.md#v200-aug-2023.  I will follow the guide to try and upgrade.  

I'm wondering if it would be faster to just run the yeoman generator and move my project files over.

**Comment by ballcoach12** [comment]
Sorry for the false alarm here.  The upgrade was pretty straightforward.  I mostly had to change my imports by appending '.js' at the end to match the module format requirements.

**Reply by msujew** [reply]
No worries, we expected some confusion in that regard. That's also why we simultaneously released 1.3.0 (which is effectively 2.0.0 but without the ESM change), so adopters have a longer grace period to upgrade to ESM.
